internal/controllers/lvmcluster/status.go

```diff
@@ -0,0 +1,161 @@
+package lvmcluster
```

Comment:
This entire file seems pretty huge for 2 simple conditions, why not use a template variable for the condition and then just deep copy this whenever you need to initialize them?